hypha/apply/funds/workflow.py

```diff
@@ -343,11 +343,12 @@ def make_permissions(edit=None, review=None, view=None):
                 'ext_internal_review': _('Open Review'),
                 'ext_determination': _('Ready For Determination'),
                 'ext_rejected': _('Dismiss'),
+                'ext_screening_withdrawn': _('Withdraw'),
```

Comment:
Why is it not enough with just `ext_withdrawn` in all places? We also need this for all workflows.

Reply:
I think the idea was that we wanted to capture the stage at which it was withdrawn, for the workflow timeline at the top of the submission.  Should we keep it like that, or just put `ext_withdrawn` in the final step (next to accepted, rejected, etc)

Reply:
We do not do that for any other state so I prefer a single *_withdraw" per workflow.

Reply:
That's fine!  Do you have a preference where in the workflow it goes?  I was going to put it at the end next to accept/reject, but I could also make an argument that it should go at the beginning (INITIAL_STATE)

Reply:
This appears completed to me.